db/migrate/20200917020736_create_followers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFollowers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :followers do |t|
+      t.references :follower_user
+      t.references :followed_user
```

Comment:
this seems a little clumsy name-wise
the table and model is referred to as "Follower"
but what it really represents is a map between a user, and the user it's following
in my opinion, the model should probably be one of: "Following", "FollowMap".
the following_user should simply be user (or user_id)
the followed_user should simply be followed (or followed_id)
i.e. a "Following" or "FollowMap" represents an instance of a user following the followed, 
god 'follow' stopped looking like a word to me

Reply:
Just thought of an alternative:
a "Follower" represents a user, and the following
user: user being followed
following: user following the user

Reply:
i'm confused by your second comment. i just changed it to user_id and followed_id